Formula/librsvg.rb

```diff
@@ -5,11 +5,12 @@ class Librsvg < Formula
   sha256 "6baf48a9d3a56fd13bbfbb9f1f76759b240b70a1fa220fd238474d66a926f98c"
   license "LGPL-2.1-or-later"
 
-  # We use a common regex because librsvg doesn't use GNOME's "even-numbered
-  # minor is stable" version scheme (at least as a "trial" for 2.55.x).
+  # librsvg doesn't use GNOME's "even-numbered minor is stable" version scheme.
+  # This regex matches any version that doesn't have a 90+ patch version, as
+  # those are development releases.
   livecheck do
     url :stable
-    regex(/librsvg[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    regex(/librsvg[._-]v?(\d+\.\d+\.(?:\d|[1-8]\d+)(?:\.\d+)*)\.t/i)
```

Comment:
IMO I think the `[1-8]` is just a hack for what we know right now.

Reply:
I personally think pagematch with the `- stable` is a better route

Reply:
There isn't any guarantee that a download.gnome.org tarball will be available at the same time that a release is created on GitLab, so I still believe that checking the `stable` source is more appropriate. Besides that, this GitLab tags page is about 3.5x the size of the download.gnome.org `cache.json` file that the `Gnome` strategy checks (162 KB vs. 45 KB).

It's pretty common for GNOME-related projects to use a 90+ patch to indicate development releases (even some non-GNOME projects use this convention), so I don't imagine we'll run into any issues by simply omitting those versions. If we do, we can always revisit this, though.